src/main/java/teammates/ui/output/StudentResponseAccessLog.java

```diff
@@ -0,0 +1,32 @@
+package teammates.ui.output;
+
+/**
+ * The response access log of a student for a single feedback response session.
+ */
+public class StudentResponseAccessLog {
+    private final StudentData studentData;
+
+    private final FeedbackResponseAccessType feedbackResponseAccessType;
+
+    private final long accessTimestamp;
+
+    public StudentResponseAccessLog(StudentData studentData,
+                                    FeedbackResponseAccessType feedbackResponseAccessType,
+                                    long accessTimestamp) {
+        this.studentData = studentData;
+        this.feedbackResponseAccessType = feedbackResponseAccessType;
+        this.accessTimestamp = accessTimestamp;
+    }
```

Comment:
Looks good to me, though it's probably down to variable namings again!
Perhaps the `FeedbackResponseAccessType` could be something like `FeedbackSessionLogType` (feels more natural to me)? Or `accessTimestamp` as simply `timestamp` and `StudentResponseAccessLog` as `FeedbackSessionLogEntry` etc.?